### DISCUSSION TITLE
How about a Wiki page explaining how to use icons?

### DISCUSSION THREAD

**Discussion by bound-variable** [ryonakano/pinit#30] [Ideas]
We could possibly make the process of pinning apps more efficient by offering an explanation of how icons work in at least Gnome and KDE, e.g., which file types are acceptable and at what sizes.

I would be happy to draft such an explanation.

**Comment by ryonakano** [comment]
Thank you for your idea! However I personally prefer showing as all info as possible, not relying on the wiki, because not everyone see the wiki out of the app. I would like to give these info like icon format in the app in some way.
